oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticFullTextAnalyzerTest.java

```diff
@@ -104,8 +104,8 @@ public void fulltextSearchWithAdvancedLanguageBasedStemmer() throws Exception {
             anl.addChild(FulltextIndexConstants.ANL_TOKENIZER).setProperty(FulltextIndexConstants.ANL_NAME, "Standard");
 
             Tree filters = anl.addChild(FulltextIndexConstants.ANL_FILTERS);
-            filters.addChild("LowerCase");
-            Tree stemmer = filters.addChild("stemmer");
+            addFilter(filters, "Lowercase");
```

Comment:
Is this typo (LowerCase --> Lowercase)?

Reply:
Good catch. It's not an issue since the filter names are lowercased when the config gets processed. I have changed it so the configuration is consistent with the rest.